.maintain/build-image.sh

```diff
@@ -22,3 +22,19 @@ docker build \
     --build-arg NODE_TYPE=standalone-node \
     -t local/standalone-node:$target_tag \
     .
+
+# build DIP provider and consumer templates
+PROVIDER_BIN_NAME="dip-provider-node-template"
+docker build \
+	--cache-from $AWS_REGISTRY/kilt-parachain/collator:builder \
```

Comment:
Based on https://github.com/KILTprotocol/kilt-node/pull/587#discussion_r1407337975, I used the same base image for caching. Is that correct @weichweich?

Reply:
LGTM